CODE_OF_CONDUCT_COMMITTEE.md

```diff
@@ -0,0 +1,242 @@
+# Guide for the Sage Code of Conduct Committee #
```

Comment:
Would it be possible to move this to the documentation (as part of a new "Governance" section), a bit similar to https://jupyter.org/governance/conduct/enforcement.html.
That would reduce the "clutter" in the root and also makes it possible to find the information via google.

Reply:
Apparently google is intelligent enough to index certain files in the git repo. I don't have any luck with "Authors of sagemath"

Reply:
I prefer to leave it where it is: at the top level and with a name all in caps, the file is very visible to people who view the code on GitHub and to people who clone the git repo. If this ticket is merged (which will require a vote on sage-devel, since it's a change to the code of conduct), then I will open a follow-up ticket to add links to the developer's guide, pointing to both the code and the committee manual. I hope that those links will help search engines find the relevant documents. How does that sound?

Reply:
> the file is very visible to people who view the code on GitHub and to people who clone the git repo

Right, but why does a manual for the code of conduct committee require such a prominent placement?

Reply:
First, why not give it prominent placement? Second, it lists possible actions that can be taken by the committee, and those can be important. When code-of-conduct issues arise, users should be able to find the relevant information quickly and easily, so prominent placement seems like a good choice.

Reply:
Also, CoCcttee's case-by-case actions and deliberations will be rather opaque to the general developer community, because privacy of those involved needs to be respected. Hopefully we can make up for that by being transparent about the guidelines and process the committee follows to perform those deliberations and come to its actions.

It could end up being a rather technical document, so if at some point we do get to a "governance" folder, the manual would be a candidate for being moved there. But then it would be there together with other important documents.

Reply:
The ongoing election of the CoCC is a good example of privacy violations by the committee.

There must be no typo's in the list of candidates, but, well...

Then, it's sensitive enough to prevent the candidates sitting on the current committee from viewing individual ballots, but this was not done, either.

Reply:
> Then, it's sensitive enough to prevent the candidates sitting on the current committee from viewing individual ballots, but this was not done, either.

Yes. But it is also hard to imagine a way to have prevented that while processing the vote openly.

Reply:
> > Then, it's sensitive enough to prevent the candidates sitting on the current committee from viewing individual ballots, but this was not done, either.
> 
> Yes. But it is also hard to imagine a way to have prevented that while processing the vote openly.
> 
> 
this is a rather standard problem, and there are free solutions for it, not requiring any 3rd trusted persons, e.g.

https://adoodle.org/

Reply:
> > > Then, it's sensitive enough to prevent the candidates sitting on the current committee from viewing individual ballots, but this was not done, either.
> > 
> > 
> > Yes. But it is also hard to imagine a way to have prevented that while processing the vote openly.
> 
> this is a rather standard problem, and there are free solutions for it, not requiring any 3rd trusted persons, e.g.
> 
> https://adoodle.org/

It would have been great if you'd suggested this option when I asked for feedback on how to run the vote.  At this point, I think the best option is to use an anonymous system like this in the future (which I would be supportive of).

Reply:
> this is a rather standard problem, and there are free solutions for it, not requiring any 3rd trusted persons, e.g.
> 
> https://adoodle.org/

well ... `adoodle.org` does have a trusted role in this. I guess you could argue it's not a person and it probably is trustworthy, but it's still a third party that gets the data and needs to be trusted to do what is intended (and hopefully nothing more).

Reply:
Advice will come in few days, due to travels. However, I was already told that simple approval voting is not a very fair voting scheme - but the story is too long to be fully done in a messenger app :-)

Reply:
@nbruin - imagine, of all of things, that people in Boston (of Boston Bruins NHL team) don't know how to spell your surname...